components/icon-button.jsx

```diff
@@ -4,18 +4,23 @@ var Link = Router.Link;
 var ImageTag = require('./imagetag.jsx');
 
 var LinkSwap = React.createClass({
+  propTypes: {
+    linkTo: React.PropTypes.string,
+    href: React.PropTypes.string,
+    onClick: React.PropTypes.func
```

Comment:
ohhhhhh I think we can utilize [`{...other}`](https://facebook.github.io/react/docs/transferring-props.html) :smile_cat: